README.org

```diff
@@ -6,16 +6,20 @@ Emacs major mode for editing [[https://xon.sh][xonshrc]] files. This mode inheri
 * Installation
 
 For now just copy the xonsh-mode.el file to your ~/.emacs.d/elisp/
-folder and then make sure the following line is executed on startup:
+folder and then make sure the following two lines are executed on
+startup:
 
 #+begin_src elisp
 (add-to-list 'load-path "~/.emacs.d/elisp/")
+(require 'xonsh-mode)
 #+end_src
 
 Which if you are using the scheme above would be in the
 ~/.emacs.d/init.el folder.
 
 
+
+
```

Comment:
Well, one more question: why the extra space?

Reply:
twas an accident, and it renders the same.